pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "django-jazzmin"
-version = "2.6.1"
+version = "3.0.0"
```

Comment:
This can also be 0.0.0, as versions are tracked outside, allowing us to quantify major/minor/patch at the point of release as opposed to the point of change.